### ISSUE TITLE
xUnit1012 when using `[InlineData(<decimal>)]`

### ISSUE THREAD

**Issue by colejohnson66** [xunit/xunit#3000]
xUnit gives a false warning when using `[InlineData(0.1m)]`:

```csharp
public sealed class ReproClass
{
    [Theory]
    [InlineData(0.1m)]
    public void ReproMethod(decimal m)
    {

    }
}
```

> xUnit1012: Null should not be used for type parameter 'm' of type 'decimal'. Use a non-null value, or convert the parameter to a nullable type.

**Comment by colejohnson66** [comment]
Yes. I am aware of that limitation for attributes.

**Comment by bradwilson** [comment]
Available in `1.16.0-pre.30` https://xunit.net/docs/using-ci-builds
